docs/specification/1.0.md

```diff
@@ -199,13 +199,15 @@ The prefetch tokens defined by this `example-hook` would be `{{context.patientId
 An EHR MAY choose to honor some or all of the desired prefetch templates, and is free to choose the most appropriate source for these data. For example:
 
 - The EHR MAY have some of the desired prefetched data already in memory, thereby removing the need for any network call
-- The EHR MAY compute an efficient set of prefetch templates from multiple CDS Services, thereby reducing the number of network calls to a minimum
+- The EHR MAY compute an efficient set of prefetch templates from multiple CDS Services, thereby reducing the number of calls to a minimum
 - The EHR MAY satisfy some of the desired prefetched templates via some internal service or even its own FHIR server.
 
 The EHR SHALL deny access if the requested resource is outside the user's authorized scope.
 
 Regardless of how the EHR satisfies the prefetch templates (if at all), the prefetched data given to the CDS Service MUST be equivalent to the data the CDS Service would receive if it were making its own call to the EHR FHIR server using the parameterized prefetch template.
 
+> Note that this means that CDS services will receive only the information they have requested and are authorized to receive. Prefetch data for other services registered to the same hook MUST NOT be provided.
```

Comment:
Would you be willing to clarity 'Prefetch data for other services registered to the same hook MUST NOT be provided.'

Do you mean unauthorized?

Reply:
It's even stronger than unauthorized. A service may be authorized to view some piece of data, but if it did not ask for it in it's prefetch, it should not receive it. So how about if we add a sentence at the end of the note:

"In other words, services should receive ALL and ONLY the data they requested in their prefetch and for which they are authorized recipients."

Reply:
In other words, services SHALL only receive the data they requested in their prefetch and for which they are authorized recipients.